tempesta_fw/apm.c

```diff
@@ -1151,14 +1496,318 @@ tfw_cfgop_apm_stats(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	return 0;
 }
 
+static int
+tfw_cfgop_apm_server_failover(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	TfwApmHMCfg *hm_entry;
+	int code, limit, tframe;
+
+	if (tfw_cfg_check_val_n(ce, 3))
+		return -EINVAL;
+
```

Comment:
My understanding is that these two checks are congenerical -  about a very similar thing, simple, and logically in the same realm. Usually there's no need for an empty line between checks like these as they are not logically separated. There's lot of examples in the code for this.